src/LibVLCSharp.Forms/Shared/MediaPlayerElement.xaml.cs

```diff
@@ -242,7 +259,189 @@ private void PageDisappearing(object sender, Page e)
 
         private void GestureRecognized(object sender, EventArgs e)
         {
+            if(EqualizerControls != null && EqualizerControls.IsVisible)
+            {
+                EqualizerControls.IsVisible = false;
+            }
             PlaybackControls.Show();
-        }        
+        }
+
+        private Frame? EqualizerControls { get; set; }
+        private Switch? EnableEqualizerSwitch { get; set; }
+        private Picker? PresetsDataPicker { get; set; }
+        private Slider? PreampSlider { get; set; }
+        private BindableStackLayout? FrequenciesLayout { get; set; }
+        private Equalizer? MediaEqualizer;
+        private Preset? SelectedPreset { get; set; }
+        private Switch? SnapBandsSwitch { get; set; }
+        private ObservableCollection<Band> AmpBands = new ObservableCollection<Band>();
+        // Workaround to prevent Slider issues
```

Comment:
can you be more specific about which issues exactly please?

Reply:
The Slider ValueChanged event(`BandAmplificationValueChanged`) triggers 2 times when it value changes. That's why I created the `IsTheSameAmplification `method in the Utility class because the newValue is the same when the event triggers at the second time. If there was a DragCompleted event for the Slider, it will be ok.

Now I used `EqualizerViewIsOpening`, `IgnoreAmpValueChanged`, `AmpSliderTriggerCount` to prevent the call of `BandAmplificationValueChanged` method when I set values to the amplifications(The ten vertical sliders).